### ISSUE TITLE
206 false positive. 

### ISSUE THREAD

**Issue by alex-left** [ansible/ansible-lint#690]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible 2.9.2

ansible-lint 4.2.0
```

- ansible installation method: pip
- ansible-lint installation method: pip

I have a line like:
```
 src: "{{ prometheus_config | default('prometheus.yml.j2') }}"
```

Ansible-lint throws an 206 but the line have right syntax an spaces.

**Comment by ssbarnea** [comment]
Unable to reproduce with master.
